.goreleaser.yml

```diff
@@ -60,6 +60,14 @@ brews:
     test: |
       help_text = shell_output("#{bin}/fastly --help")
       assert_includes help_text, "Usage:"
+scoop:
+  bucket:
+    owner: fastly
+    name: scoop-cli
+  homepage: https://github.com/fastly/cli
+  skip_upload: auto
```

Comment:
this setting seems to only apply to homebrew, according to the docs, though perhaps they are missing this field for scoop.